optimum/intel/openvino/configuration.py

```diff
@@ -103,6 +103,14 @@
     },
 }
 
+_DEFAULT_4BIT_CONFIG = {
+    "bits": 4,
+    "ratio": 1.0,
+    "sym": False,
+    "group_size": 128,
```

Comment:
I'm not sure about this value, because some models can has smaller head_dim

Reply:
Yes, `group_size` was previously `None`, but I decided to make it more explicit and set it to 128. Otherwise, the decision will be left to be made by NNCF. If `group_size=None` is provided, NNCF will set it to 128 too. But this can potentially change in the future in theory resulting in some bad chain reaction. This is was my motivation.

> some models can has smaller head_dim

This is true. But as I understand, it will fail even if `None` is provided for the reason described above.